android/assets/jsons/translations/Ukrainian.properties

```diff
@@ -810,23 +810,23 @@ A(n) [nukeType] exploded in our territory! = [nukeType] вибухнула на
 After being hit by our [nukeType], [civName] has declared war on us! = Скинувши [nukeType] на [civName], ми змусили їх оголосити нам війну
 The civilization of [civName] has been destroyed! = [civName]: цивілізація знищена!
 The City-State of [name] has been destroyed! = [name]: місто-держава знищено!
-Your [ourUnit] captured an enemy [theirUnit]! = Ваш підрозділ [ourUnit] узяв у полон ворожий підрозділ [theirUnit]!
-Your [ourUnit] plundered [amount] [Stat] from [theirUnit] = Ваш підрозділ [ourUnit] награбував [amount] [Stat] у [theirUnit]
+Your [ourUnit] captured an enemy [theirUnit]! = Ваш підрозділ «[ourUnit]» узяв у полон ворожий підрозділ «[theirUnit]»!
+Your [ourUnit] plundered [amount] [Stat] from [theirUnit] = Ваш підрозділ «[ourUnit]» награбував [amount] [Stat] у [theirUnit]
 We have captured a barbarian encampment and recovered [goldAmount] gold! = Ми захопили табір варварів і отримали золота: [goldAmount]
 An enemy [unitType] has joined us! = Ворожий підрозділ [unitType] приєднався до нас!
 
 # This might be needed for a rewrite of Germany's unique - see #7376
-A barbarian [unitType] has joined us! = Варварський підрозділ [unitType] приєднався до нас!
+A barbarian [unitType] has joined us! = Варварський підрозділ «[unitType]» приєднався до нас!
 
 We have found survivors in the ruins - population added to [cityName] = У руїнах знайдено вцілілих — вони приєдналися до населення міста [cityName]
 We have discovered cultural artifacts in the ruins! (+20 Culture) = У руїнах знайденi культурнi цiнностi! (культура +20)
 We have discovered the lost technology of [techName] in the ruins! = У руїнах знайдено загублену технологію [techName]!
 A [unitName] has joined us! = До нас приєднався підрозділ [unitName]!
-An ancient tribe trains our [unitName] in their ways of combat! = Давнє племʼя навчило їхнього способу битви наш підрозділ [unitName]!
+An ancient tribe trains our [unitName] in their ways of combat! = Давнє племʼя навчило їхнього способу битви наш підрозділ «[unitName]»!
 We have found a stash of [amount] gold in the ruins! = У руїнах знайдено скарб золота кількістю [amount]!
 We have found a crudely-drawn map in the ruins! = Ми знайшли в руїнах грубо намальовану мапу!
-[unit] finished exploring. = Підрозділ [unit] завершив розвідку.
-[unit] has no work to do. = Підрозділ [unit] без роботи.
+[unit] finished exploring. = Підрозділ «[unit]» завершив розвідку.
+[unit] has no work to do. = [unit] без роботи.
```

Comment:
Why no quotes here, and inconsistent translation even comparing to the line above?

Reply:
The primary problem of units: the gender of the unit can be different. 
In the first case we resolve this by referencing it as always-male "підрозділ завершив" and the name/type of the unit goes in quotes. 
In the second case there is no need to resolve that: any gender will be suitable since we have no verb there. For example, Робітник без роботи, Робітниця без роботи, Бо-зна-що без роботи.

Reply:
Got it, thanks